lte/gateway/c/core/oai/test/amf/amf_app_test_util.cpp

```diff
@@ -164,6 +160,217 @@ int send_uplink_nas_registration_complete(
   return (rc);
 }
 
+/* Create pdu session establishment  request from ue */
+int send_uplink_nas_pdu_session_establishment_request(
+    amf_app_desc_t* amf_app_desc_p, amf_ue_ngap_id_t ue_id, const plmn_t& plmn,
+    const uint8_t* nas_msg, uint8_t nas_msg_length) {
+  bstring pdu_session_est_req;
+  tai_t originating_tai = {};
+
+  if ((!amf_app_desc_p) || (!nas_msg) || (nas_msg_length == 0)) {
+    return RETURNerror;
+  }
+
+  pdu_session_est_req = blk2bstr(nas_msg, nas_msg_length);
+
+  originating_tai.plmn = plmn;
+  originating_tai.tac  = 1;
+
+  ue_m5gmm_context_s* ue_context_p =
+      amf_ue_context_exists_amf_ue_ngap_id(ue_id);
+  if (!ue_context_p) {
+    return RETURNerror;
+  }
```

Comment:
This can also be removed from here

Reply:
accessing ue_context_p pointer variable to fetch apn config profile.
> apn_config_profile_t& profile = ue_context_p->amf_context.apn_config_profile;

 if fetching ue context fails, the system should not crash, error reporting is ideal behaviour.